answers/models.py

```diff
@@ -47,6 +48,10 @@ def set_notes(self, notes):
     def get_notes(self):
         return self.response
 
+    def update(self, new_answer):
+        self.text = new_answer
+        self.save()
+
     class Meta:
```

Comment:
Are Answers ever ordered anywhere right now? We might want to add an ordering to the meta class to prevent any UI jank (can probably just order by `created_on`).

Reply:
Done.